### ISSUE TITLE
aws_vpc secondary cidr block

### ISSUE THREAD

**Issue by benrobinsonsonos** [terraform-providers/terraform-provider-aws#3403]
### Affected Resource(s)
- aws_vpc

### Terraform Configuration Files
```hcl
resource "aws_vpc" "vpc" {
  cidr_block = "10.0.0.0/16"
  secondary_cidr_blocks = ["10.2.0.0/16"]
}
```
### Expected Behavior
Attach secondary CIDR block to VPC.

### Actual Behavior
secondary_cidr_blocks is not a supported argument for resource aws_vpc.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

### Important Factoids
AWS announced support for secondary cidr blocks in August 2017. Argument should accept a list of cidr blocks.

### References
https://aws.amazon.com/about-aws/whats-new/2017/08/amazon-virtual-private-cloud-vpc-now-allows-customers-to-expand-their-existing-vpcs/

https://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_Subnets.html#vpc-resize

**Comment by ewbankkit** [comment]
Implemented in https://github.com/terraform-providers/terraform-provider-aws/pull/1568.

**Comment by bflad** [comment]
A new `aws_vpc_ipv4_cidr_block_association` resource (instead of inline on the `aws_vpc` resource) has been merged into master and will be released with version 1.26.0 of the AWS provider, likely middle of next week. 🎉

**Comment by Hitesh-Agrawal** [comment]
Waiting for this feature.
